### PR TITLE
home-add-page-25.06.23

### PR DESCRIPTION
**Pull request by nicitaacom**

### CODE REVIEW

**Comment by nicitaacom** [comment]
Review my code so you can understand wha's going on
I wrote question in dicord but it sill not answered - that's why I created this pull request

**Comment by ottakist** [comment]
It looks good on the mobile version but pretty bad on the desktop(not nearly close as in Figma design).
Do you want to make a pull request for now and fix it in later pull requests?

**Comment by ottakist** [comment]
image
The "Top sales" section doesn't really look  as in the Figma design 
image

**Comment by ottakist** [comment]
and the text in this section is too big. You basically don't have space for padding. Without padding it's too heavy and squeezed 
image

**Comment by ottakist** [comment]
Overall, it's fine, I guess. The mobile version needs some improvements.

**Comment by ottakist** [comment]
About the desktop version... Yeah there is a lot of work to do

**Comment by nicitaacom** [comment]
> It looks good on the mobile version but pretty bad on the desktop(not nearly close as in Figma design). Do you want to make a pull request for now and fix it in later pull requests?

Yes - as you said we need to get full overview of our project and then improve something
Also I don't finish it because I fint it necesarry to be up to date with 'development' branch because if I spend 1-2d to fix it during this time I add some components and other will not have those components - that's why I did this pull request
Absolutely agreed about desktop version

**Comment by nicitaacom** [comment]
Have you access to merge it into 'development' branch - or I do it insted of you?
If you have access - do it

File: src/components/Container.tsx
Comment (ottakist) [inline src/components/Container.tsx:1]: why are you ignoring this issue?
image
if you have a value that could be null you should at least provide a default value like this:
image
if you don't want to see "null" in your classList

**Review by ottakist — APPROVED** [review]

**Pull request merged**